float32 num1
float32 num2
string opp
---
bool success
float32 value